.github/workflows/daily_reports.yaml

```diff
@@ -11,53 +11,62 @@ env:
 
 jobs:
   run-hive:
-    name: Hive - ${{ matrix.name }}
+    name: Hive (${{ matrix.vm }}) - ${{ matrix.test.name }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        include:
-          - name: "Rpc Compat tests"
-            file_name: rpc-compat
-            simulation: ethereum/rpc-compat
-          - name: "Devp2p eth tests"
-            file_name: devp2p
-            simulation: devp2p
-          - name: "Cancun Engine tests"
-            file_name: engine
-            simulation: ethereum/engine
-          - name: "Sync tests"
-            file_name: sync
-            simulation: ethereum/sync
+        vm: [levm, revm]
+        test:
+          - {
+              name: "Rpc Compat tests",
+              file_name: rpc-compat,
+              simulation: ethereum/rpc-compat,
+            }
+          - { name: "Devp2p eth tests", file_name: devp2p, simulation: devp2p }
+          - {
+              name: "Cancun Engine tests",
+              file_name: engine,
+              simulation: ethereum/engine,
+            }
+          - { name: "Sync tests", file_name: sync, simulation: ethereum/sync }
 
     steps:
       - name: Pull image
+        if: ${{ matrix.vm == 'revm' }}
         run: |
           docker pull ghcr.io/lambdaclass/ethrex:latest
           docker tag ghcr.io/lambdaclass/ethrex:latest ethrex:latest
 
       - name: Checkout sources
         uses: actions/checkout@v4
 
+      - name: Build Image with LEVM
```

Comment:
TODO:  we probably want to upload a build with levm to docker, like we do with revm